problem installing grayflow: cannot find -lX11

Hi
I'm trying to compile grayflow from source on centos4
after entering `make` command I get the following error:
`Linking C shared library libycadgraywolf.so
/usr/bin/ld: cannot find -lX11
collect2: ld returned 1 exit status
make[2]: *** [src/Ylib/libycadgraywolf.so.1.0.0] Error 1
make[1]: *** [src/Ylib/CMakeFiles/ycadgraywolf.dir/all] Error 2
make: *** [all] Error 2
`
How can I fix it?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,7 @@ cmake_minimum_required (VERSION 2.6)
 project (graywolf)
 
 find_package(PkgConfig)
+find_package(X11)
 INCLUDE(CheckIncludeFiles)
 
 

--- a/src/Ylib/CMakeLists.txt
+++ b/src/Ylib/CMakeLists.txt
@@ -5,11 +5,11 @@
 
 add_library(ycadgraywolf SHARED assign.c buster.c cleanup.c colors.c deck.c dialog.c draw.c dset.c edcolors.c file.c getftime.c graph.c grid.c hash.c heap.c list.c log.c menus.c message.c mst.c mytime.c okmalloc.c path.c plot.c program.c project.c queue.c quicksort.c radixsort.c rand.c rbtree.c relpath.c set.c stat.c stats.c string.c system.c time.c timer.c trans.c wgraphics.c ydebug.c yreadpar.c )
 
-target_link_libraries(ycadgraywolf X11)
+target_link_libraries(ycadgraywolf ${X11_LIBRARIES})
 target_link_libraries(ycadgraywolf m)
 
 set_target_properties(ycadgraywolf PROPERTIES VERSION 1.0.0 SOVERSION 1)
 
-INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR}/include ${CMAKE_BINARY_DIR}/include)
+INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR}/include ${CMAKE_BINARY_DIR}/include ${X11_INCLUDE_DIR})
 
 install(TARGETS ycadgraywolf DESTINATION lib)

